src/router/index.js

```diff
@@ -31,7 +31,15 @@ const routes = [
   {
     path: "/lists",
     name: "lists",
-    component: () => import("../views/Lists")
+    component: () => import("../views/Lists"),
+    meta: {
+      requiresAuth: true
+    }
+  },
```

Comment:
> As a anonymous user

#51 The list viewing page should be available for unauthenticated users.

Reply:
:+1: